runtime/doc/remote_plugin.txt

```diff
@@ -108,7 +108,9 @@ every time a remote plugin is installed, updated, or deleted.
 Vimscript file containing declarations for all Vimscript entities
 (commands/autocommands/functions) defined by all remote plugins, with each
 entity associated with the host and plugin path. The manifest is a generated
-extension to the user's vimrc (it even has the vimrc filename prepended).
+extension to the user's |vimrc| (it even has the vimrc filename prepended).
```

Comment:
Yes if the vimrc is found, but I wasn't sure how to word it without getting too technical when the vimrc isn't found.